### PR TITLE
Update documentation links

### PR DESCRIPTION
**Pull request by marcospereira**
## Purpose

Correct, update and remove some documentation links.

### CODE REVIEW

**Comment by ignasi35** [comment]
Uh, oh! The docs validation failed:

```

SLF4J: Actual binding is of type [org.slf4j.impl.SimpleLoggerFactory]
[error] Could not find resource manual/hacking/images/TravisCI-Full-Color.png at /home/travis/build/playframework/playframework/documentation/manual/hacking/ThirdPartyTools.md:8
[error] [[images/TravisCI-Full-Color.png]]
[error]   ^
java.lang.RuntimeException: Documentation validation failed
	at com.typesafe.play.docs.sbtplugin.PlayDocsValidation$$anonfun$16.apply(PlayDocsValidation.scala:451)
	at com.typesafe.play.docs.sbtplugin.PlayDocsValidation$$anonfun$16.apply(PlayDocsValidation.scala:324)
```

**Comment by marcospereira** [comment]
@ignasi35 adding the image. :-)

File: documentation/manual/working/commonGuide/assets/AssetsCoffeeScript.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Here is an example layout for using CoffeeScript in your projects:
 ```
 app
  └ assets
-    └ javascripts
+    └ javascriptsR
```
Comment (PromanSEW) [inline documentation/manual/working/commonGuide/assets/AssetsCoffeeScript.md]: Typo?

Comment (marcospereira) [reply documentation/manual/working/commonGuide/assets/AssetsCoffeeScript.md]: Fixed. Thanks, @PromanSEW.

**Comment by ignasi35** [comment]
This didn't backport cleanly so I created a separate PR: https://github.com/playframework/playframework/pull/8653

**Pull request merged**